### PR TITLE
+ Add test subscription webhook for Android

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: modmod/views/membership.py [hunk 1]
```diff
@@ -311,11 +317,83 @@ def post_android_subscription(request):
...
+        'message': 'ok',
+    }
+
+    if 'subscriptionNotification' not in data:
+        return default_response
+
+    subscription = data['subscriptionNotification']
+    notification_type = subscription['notificationType']
+
+    log.info('RECEIVE_ANDROID_SUBSCRIPTION_WEBHOOK_TYPE_' + str(notification_type))
```
Comment (smlam2048) [inline modmod/views/membership.py]: 這些 log 是測試，還是有用的？

Comment (nwingt) [reply modmod/views/membership.py]: For Stackdriver logging

File: modmod/views/membership.py [hunk 2]
```diff
@@ -283,13 +289,13 @@ def post_android_subscription(request):
...
         developer_payload = response_dict['developer_payload']
         payout_amount = int(math.ceil(get_iap_sub_price() * get_iap_sub_price_payout_ratio() * 100))/100.0
-        result = UserOperations.handle_membership_update(user,
-                                                         product_id,
-                                                         original_transaction_id,
-                                                         expire_date,
-                                                         developer_payload,
-                                                         "android",
-                                                         payout_amount)
+        UserOperations.handle_membership_update(user,
```
[the remark this answers is not in the document]
Comment (nwingt) [reply modmod/views/membership.py]: 沒影響到iOS

[the remark this answers is not in the document]
Comment (smlam2048) [reply modmod/views/membership.py]: user.py `handle_membership_update` return 'ok' 改了做 user.
result 不會是 'ok':
https://github.com/likecoin/oice-server/blob/1aac21e1f49d669c2fcfa6a572d3c313f13cc0e3/modmod/views/membership.py#L467

[the remark this answers is not in the document]
Comment (nwingt) [reply modmod/views/membership.py]: Updated

**Pull request merged**